src/remote/card-service.ts

```diff
@@ -38,6 +39,6 @@ export async function createCard(
  */
 
 export async function getCards() {
```

Comment:
- Function should be getSets to hit sets endpoint
- Currently this function gets all sets, need to specify ids, etc